slybot/slybot/plugins/selectors/__init__.py

```diff
@@ -0,0 +1,28 @@
+
+class Selectors(object):
+    def setup_bot(self, settings, spec, items, extractors):
+        self.selectors = {} # { item_type: { field_name: {..} }
+
+        for template in spec['templates']:
+            selectors = self.selectors.setdefault(template['scrapes'], {})
+            selectors.update(template.get('selectors', {}))
```

Comment:
Yes, selectors could be overwritten if there are set in more than one template that extracts the same item type.

Still, I think several selectors for one field is overkill, since CSS and xpath already have a way to do a union of several selectors (`//summary | //h1` and `summary, h1`). What do you think would be the best solution?
- We could put the item definitions in the item instead and all templates that extract the same item use that selectors
- Keep the selectors in the template definition and run only that selectors only when that template is the one that matches
- Keep it like it is now (merge the selectors of every template that extracts the same item and run all of them when any template extracts an item)